uaclient/cli.py

```diff
@@ -286,6 +286,8 @@ def action_detach(args, cfg):
         print("Detach will disable the following service{}:".format(suffix))
         for ent in to_disable:
             print("    {}".format(ent.name))
+    if not util.prompt_for_confirmation():
+        return 0
```

Comment:
I *think* we should exit 1 here on 'n' as apt does similarly

Do you want to continue? [Y/n] n
Abort.
csmith@uptown:~$ echo $?
1